_data/sites.json

```diff
@@ -1530,6 +1530,16 @@
         ]
     },
 
+    {
+        "name": "Bloomingdales",
+        "url": "https://www.bloomingdalesprivacyportal.com/consumer/index",
+        "difficulty": "hard",
+        "notes": "Only residents of California and Nevada are allowed to make official data removal requests. However, if you contact Customer Service via one of the methods listed at https://customerservice-bloomingdales.com/articles/contact-us, they may submit a 30-day account deletion request for you anyway.",
```

Comment:
```suggestion
        "url": "https://customerservice-bloomingdales.com/articles/contact-us",
        "difficulty": "hard",
        "notes": "Only residents of California and Nevada are allowed to make official data removal requests. However, if you contact Customer Service via one of the methods listed, they may submit a 30-day account deletion request for you anyway.",
```

Unless the link there was useful for California residents, I'd vouch to make the contact form the primary link, simplifying the notes .

Reply:
Sounds good to me! NJ resident here, so I couldn't vouch for the helpfulness of that link :(